internal/provider/resource_vdb_group.go

```diff
@@ -35,18 +35,38 @@ func resourceVdbGroup() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"tags": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
We will need computed:true here, as without that, it will detect UIPs in case of auto-tagging.

Reply:
Done